### PR TITLE
chore: [IA-587] Add Zendesk analytics

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR adds the track the events that happens in the Zendesk request workflow.

## List of changes proposed in this pull request
The following events has been tracked:

| Eventi name | Attributes | Description |
| ------------- | ------------- | ------------- |
| ZENDESK_SUPPORT_COMPLETED  |  | the service request workflow has been completed. Either a new ticket has been opened, or the user has entered the list of open tickets, or the user has gone to an internal redirect link, or the user has exited screen panic mode  |
| ZENDESK_SUPPORT_CANCEL  |   | the X button was pressed in the help center screen |
| GET_ZENDESK_CONFIG_REQUEST  |  | remote configuration of Zendesk was required  |
| GET_ZENDESK_CONFIG_SUCCESS  |  | the remote configuration of Zendesk was successfully recovered  |
| ZENDESK_SUPPORT_START  | **isAssistanceForPayment**: true if it starts from a failed payment, a failed payment detail or a failed credit card entry attempt detail  | assistance was requested starting from a failed payment, a failed payment detail or a failed credit card entry attempt detail |
| ZENDESK_SUPPORT_FAILURE  | **reason**: the failure reason. Can be one of these values: `Config undefined`, `The config has no categories`, `The category has not been selected`, `The selected category has no sub-categories` | An unexpected error has occurred |
| ZENDESK_SELECTED_CATEGORY  | **category**: one of the possible category values.  | the user has chosen the category for which he wants assistance |
| GET_ZENDESK_CONFIG_FAILURE  | **reason**: the reason for the failure | An attempt to retrieve the zendesk remote configuration failed  |

### CODE REVIEW

**Review by fabriziofff — CHANGES_REQUESTED** [review]
Could you also add the new events to https://docs.google.com/spreadsheets/d/1bXNRcTDKU-5l2RqCcwSbT7Y1M9QQTWtS_f-3-AIqSYk/edit#gid=0 ?

File: ts/features/zendesk/analytics/index.ts
Comment (debiff) [reply ts/features/zendesk/analytics/index.ts:23]: Sure, modified in [2ed5c8f](https://github.com/pagopa/io-app/pull/3601/commits/2ed5c8f91e71588568e40a9635102a8f58e9067f)

File: ts/features/zendesk/screens/ZendeskChooseCategory.tsx
Comment (debiff) [reply ts/features/zendesk/screens/ZendeskChooseCategory.tsx]: Yes, thank you. Updated in [09f6a19](https://github.com/pagopa/io-app/pull/3601/commits/09f6a19be9e7ff9d74f93b5a181d7a1b60974146)

**Comment by debiff** [comment]
> Could you also add the new events to https://docs.google.com/spreadsheets/d/1bXNRcTDKU-5l2RqCcwSbT7Y1M9QQTWtS_f-3-AIqSYk/edit#gid=0 ?

Sure! I was waiting for the legal approval, I'll add them right away!

**Review by fabriziofff — APPROVED** [review]

**Pull request merged**
